jenkins/clean_project.py

```diff
@@ -69,10 +69,10 @@ def main(project, days, filt, delete):
       description='Delete old instances from a project')
   parser.add_argument('--project', help='Project to clean', required=True)
   parser.add_argument(
-      '--days', type=float, help='Clean items more than --days old', required=True)
+      '--hours', type=float, help='Clean items more than --hours old', required=True)
   parser.add_argument(
       '--delete', action='store_true', help='Really delete things when set')
   parser.add_argument(
       '--filter', default='name:tmp*', help='Filter down to these instances')
   args = parser.parse_args()
-  main(args.project, args.days, args.filter, args.delete)
+  main(args.project, args.hours, args.filter, args.delete)
```

Comment:
`--days=0.125` is a float, do we really need to switch to `--hours`?

I think in most maintenance scenarios `--days=14` is easier to understand than `--hours=336`

Alternatively how about having both?  `args.hours + args.days * 24`